core/src/jsMain/kotlin/io/islandtime/format/DateFormatSymbols.kt

```diff
@@ -0,0 +1,185 @@
+package io.islandtime.format
+
+import io.islandtime.intl.DateTimeFormat
+import io.islandtime.locale.MLocale
+import io.islandtime.locale.defaultLocale
+import io.islandtime.objectOf
+import kotlin.js.Date
+
+class DateFormatSymbols private constructor(
```

Comment:
Shouldn't need a `DateFormatSymbols` class. It seems like this is over-complicated by reusing parts of the JVM implementation of `PlatformDateTimeTextProvider` that really shouldn't be copied. Needs to be thoroughly refactored, I think.

Reply:
Are you suggesting refactoring IslandTime common code to avoid this?

Reply:
No -- `DateFormatSymbols` isn't part of the common code. That's a Java API that just doesn't need to be mimicked in the JS implementation.